### ISSUE TITLE
[FEAT]: Do ablation study on Mistral-7B model on hyper-parameter

### ISSUE THREAD

**Issue by lwaekfjlk** [sotopia-lab/sotopia-pi#51]
### Description

We need to test whether different learning rate or batch size matters or not.
